### PR TITLE
feat: add `rhacs_expired_at` to centrals overview panel

### PR DESCRIPTION
**Pull request by 0x656b694d**
3 commits:
1. JSON indentation;
2. New column;
3. `make generate`.

image

### CODE REVIEW

**Comment by stehessel** [comment]
Can you try to adjust the indentation of the source json such that the diff will hopefully be smaller?

**Comment by 0x656b694d** [comment]
> Can you try to adjust the indentation of the source json such that the diff will hopefully be smaller?

Please look at the [second](https://github.com/stackrox/rhacs-observability-resources/pull/199/commits/715eb551794c90ee252a0153cf6eac339f30783d) commit of the PR.

**Comment by stehessel** [comment]
> The indentation of the original JSON is absolutely unsupportable.
??? Why is that?

**Comment by 0x656b694d** [comment]
> > The indentation of the original JSON is absolutely unsupportable.
> 
> Why is that?

A mess of tabs and spaces, one cannot just copy-paste a json snippet from Grafana to patch the file and keep formatting consistent.

image

File: resources/grafana/sources/rhacs-cluster-overview.json [hunk 1]
```diff
@@ -1,1884 +1,1935 @@
...
+                    "range": false,
+                    "refId": "A"
+                },
+                {
+                    "datasource": {
+                        "type": "prometheus",
+                        "uid": "PBFA97CFB590B2093"
+                    },
+                    "editorMode": "code",
+                    "expr": "sum(process_cpu_seconds_total{namespace=~\"rhacs-$instance_id\", job=~\"central\"}) by (namespace, rhacs_expired_at)",
```
Comment (stehessel) [inline resources/grafana/sources/rhacs-cluster-overview.json:1340]: I don't think we need `sum(process_cpu_seconds_total)` three times. Can you combine the labels into a single query?

**Comment by stehessel** [comment]
I'm concerned that the next PR will once again change the indentation. Let's just add a prettier formatter then.

**Comment by stehessel** [comment]
Btw for reviewing it can be nice to save the updated dashboard under a new name and share the link. The dashboard is not persisted and will be removed once the grafana pod is restarted.

**Comment by 0x656b694d** [comment]
> I'm concerned that the next PR will once again change the indentation. Let's just add a prettier formatter then (in other PR).

@stehessel , See #200.

**Comment by 0x656b694d** [comment]
> Btw for reviewing it can be nice to save the updated dashboard under a new name and share the link. The dashboard is not persisted and will be removed once the grafana pod is restarted.

Please see the modified panel here:
https://grafana-route-rhacs-observability.apps.acs-prod-dp-01.pnz3.p1.openshiftapps.com/d/c4972b96-ea0f-45a9-83ee-3a74e5753edb/pr-199?orgId=1

File: resources/grafana/sources/rhacs-cluster-overview.json (hunk 1 above)
Comment (0x656b694d) [reply resources/grafana/sources/rhacs-cluster-overview.json]: Done, please look at the updated test dashboard.

**Review by stehessel — APPROVED** [review]
Looks good thanks. One suggestion, I played with the test dashboard a bit and found that switching `Minimum column width` from 200 -> auto renders the columns a bit nicer for me.

**Comment by 0x656b694d** [comment]
> `Minimum column width` from 200 -> auto renders the columns a bit nicer for me.

200 is the minimum width for "auto" resizing (they're two different settings). If I set less than 200, the "Expired At" column is cut for me.

**Pull request merged**